src/Components/Samples/BlazorServerApp/Pages/_Host.cshtml

```diff
@@ -13,9 +13,7 @@
     <link href="css/site.css" rel="stylesheet" />
 </head>
 <body>
-    <app>
-        @(await Html.RenderComponentAsync<App>(RenderMode.ServerPrerendered))
-    </app>
+    <component type="typeof(App)" render-mode="ServerPrerendered" />
```

Comment:
Should we make the tag helper enforce use as a self-closing tag?

The reason why I'm asking is that someone might do:
```razor
<component ... >
  some stuff
</component>
```

And have some kind of expectations about what that means - currently it means nothing, and we can make that explicitly an error.